lib/series_arithmetic.rb

```diff
@@ -119,7 +121,9 @@ def percentage_change
     new_series_data = {}
     last = nil
     data.sort.each do |date, value|
-      new_series_data[date] = (value-last)/last*100 unless last.nil?
+      if last
+        new_series_data[date] = last == 0 ? 0 : (value - last) / last * 100
```

Comment:
If last is 0 and value is 0, then 0 is the correct response.
What's missing here is that if last is 0 and value is not 0, then the percent change is undefined (should be nil)